include/STREAM/Equations/ConfinementTime.hpp

```diff
@@ -9,14 +9,13 @@
 #include "DREAM/Constants.hpp"
 #include "STREAM/Grid/EllipticalRadialGridGenerator.hpp"
 #include "STREAM/Equations/ConnectionLength.hpp"
-
-
-
-/** 
- * Implementation of the particle confinement time
- */
+#include <array>
+#include <functional>
+#include "STREAM/Equations/Coefficients.hpp"
 
 namespace STREAM{
+		class EquationSystem;
+		
         class ConfinementTime {
```

Comment:
if this is anew class, it needs to be described in detail. What does it do? Inputs/outputs/methods. At least a rough introduction...

Reply:
It is a declaration to avoid circular definition. It is defined in its header and source file